cmd/repo-updater/repos/store.go

```diff
@@ -26,13 +26,21 @@ type Store interface {
 	ListExternalServices(context.Context, StoreListExternalServicesArgs) ([]*ExternalService, error)
 	UpsertExternalServices(ctx context.Context, svcs ...*ExternalService) error
 
-	InsertRepos(context.Context, ...*Repo) error
 	ListRepos(context.Context, StoreListReposArgs) ([]*Repo, error)
-	DeleteRepos(ctx context.Context, ids ...api.RepoID) error
 	UpsertRepos(ctx context.Context, repos ...*Repo) error
 	UpsertSources(ctx context.Context, inserts, updates, deletes map[api.RepoID][]SourceInfo) error
 	SetClonedRepos(ctx context.Context, repoNames ...string) error
 	CountNotClonedRepos(ctx context.Context) (uint64, error)
+
+	// EnqueueSyncJobs enqueues sync jobs per external service where their next_sync_at time
+	// is in the past.
```

Comment:
```suggestion
	// EnqueueSyncJobs enqueues sync jobs per external service where their next_sync_at is due.
```

This seems more accurate because if `next_sync_at` is the exact current moment, the job should also be enqueued?

i.e. On line 1059: `next_sync_at <= clock_timestamp()`